site/src/sphinx/server-annotated-service.rst

```diff
@@ -318,7 +335,7 @@ If your exception handler is not able to handle a given exception, you can call
         }
     }
 
-To catch an exception from every method in your service object, annotate your class with the annotation.
+To catch an exception from every method in your service class, annotate your class with the annotation.
```

Comment:
How about:

    You can annotate at class level to catch an exception from every method in your service class.